src/components/Adyen/Adyen.js

```diff
@@ -131,7 +135,14 @@ const Adyen = ({
   };
 
   const createSession = async () => {
-    const { responseData } = await createPaymentSession(isMyAccount);
+    const returnURLs = {
+      checkout: adyenConfiguration?.checkoutReturnUrl || 'https://cleeng.com',
+      myAccount: adyenConfiguration?.myaccountReturnUrl || 'https://cleeng.com'
+    };
+    const { responseData } = await createPaymentSession(
+      isMyAccount,
+      returnURLs
```

Comment:
Would it be simpler if we will pass here `adyenConfiguration?.checkoutReturnUrl`and add a logic of 'default' URL in the API file?